From: Laurent Bigonville <bigon@debian.org>
Date: Sun, 25 Sep 2022 09:57:37 +0200
Subject: configure.ac: Avoid embedding the date in the version.

Use the version from the last debian/changelog entry, otherwise the
build will differ if built on a different year and week and from git
builds vs. builds from source tarball.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6e00c0c..1daaffc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_INIT([plymouth],
-        m4_esyscmd_s([date +%y.%V.$(git rev-list $(git describe --abbrev=0)..HEAD --count) || echo 0]),
+        m4_esyscmd_s([dpkg-parsechangelog --show-field Version | sed -e 's/-[^-]*$//' | sed -e 's/^[0-9]*://']),
         [https://gitlab.freedesktop.org/plymouth/plymouth/issues])
 AC_CONFIG_SRCDIR(src/main.c)
 AC_CONFIG_HEADER(config.h)
